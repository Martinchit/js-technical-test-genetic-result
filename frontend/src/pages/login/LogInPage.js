import React from 'react';
import { LogInForm } from './components/logInform/LogInForm';
import { FlexBox } from '../../core/components/atoms/FlexBox';
import { Text } from '../../core/components/atoms/Text';

export const LogInPage = () => {
  return (
    <FlexBox padding="10px 0">
      <FlexBox
        $elevation
        width="600px"
        height="50%"
        flexDirection="column"
        backgroundColor="white"
        justifyContent="center"
        padding="5%"
      >
        <Text fontWeight="bold" fontSize={1.5} fontColor="black">
          Log In
        </Text>
        <LogInForm />
      </FlexBox>
    </FlexBox>
  );
};
