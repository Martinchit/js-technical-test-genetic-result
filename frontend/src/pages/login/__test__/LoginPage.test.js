import React from "react";
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { LogInPage } from "../LogInPage";

describe("LogInPage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<LogInPage />)
    expect(wrapper).toMatchSnapshot()
  })
})