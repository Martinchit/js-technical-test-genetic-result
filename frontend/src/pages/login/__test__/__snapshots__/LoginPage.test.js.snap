// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogInPage renders correctly 1`] = `ShallowWrapper {}`;
